### DISCUSSION TITLE
Clicking ⚙→Settings on a field layout tab should focus on the Name field in the overlay sheet

### DISCUSSION THREAD

**Discussion by heymarkreeves** [craftcms/cms#11498] [Ideas] [answered]
Each time I edit the initial Field Layout tab on a new section, I click the gear, and then Settings to change its name. It'd be great if the (required) Name field in the overlay sheet would receive focus when I do so.

**Comment by brandonkelly** [comment, marked as answer]
This actually should have already been the case! However it wasn’t working for fields due to that “Hide” checkbox which was causing problems. Just fixed for the next release (07af564fb53dd3ed3b055d6cc277ee96abd4b5e1).

**Reply by brandonkelly** [reply]
Craft 3.7.46 and 4.0.6 have been tagged with that fix.
